forms/publishdialog.ui

```diff
@@ -486,11 +509,32 @@
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
+   <widget class="QLabel" name="label_17">
+    <property name="geometry">
+     <rect>
+      <x>10</x>
+      <y>80</y>
```

Comment:
If the previous suggestions are accepted, following the proportion of 34 pixels, this value should be 78.
And finally, the `y` value of line 495 should be 44.